portfolio/src/main/webapp/blog.css

```diff
@@ -0,0 +1,98 @@
+body {
+  margin: auto;
+  border: 1px solid green;
+  padding: 4vh 2.5vw 4vh 2.5vw;
+  font-family: 'Montserrat', serif;
+}
+
+#article {
+  border: 1px solid gold;
+  padding: 1.5vh .9375vw 1.5vh .9375vw;
+}
+
+.header {
+  border: 1px solid black;
+}
+
+.title {
+  margin-top: 1vh;
+  border: 1px solid purple;
+  width: 100%;
+}
+
+.text {
+  font-size: 24pt;
+}
+
+.image {
+  height: 66vh;
+}
+
+.image img {
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+}
+
+.date {
+  border: 1px solid red;
+  font-size: 14pt;
+  width: 100%;
+}
+
+.para {
+  font-size: 24pt;
+  font-family: 'Open Sans', serif;
+}
+
+.signature {
+  border: 1px solid skyblue;
+  width: 95%;
```

Comment:
instead of the width here, are you looking to offset from something to the right? if so, adding padding is a little less error-prone.